pyswagger/tests/v2_0/test_io.py

```diff
@@ -16,19 +16,21 @@ def setUpClass(kls):
 
     def test_response_schema_with_status(self):
         """ make sure schema works as expected """
-        resp = io.SwaggerResponse(self.app.s('/resp').get)
 
         # make sure exception raised
+        resp = io.SwaggerResponse(self.app.s('/resp').get)
         self.assertRaises(Exception, resp.apply_with, None, 'some string')
 
         # test for 'default'
-        resp.apply_with(0, 'test string')
+        resp = io.SwaggerResponse(self.app.s('/resp').get)
```

Comment:
Response object is not reusable as the header is parsed with 'append' fashion, while mime codec uses response header 'content-type' for unmarshall. Hence, new response is created for each sub test case.